src-ghc/Pact/PersistPactDb/Regression.hs

```diff
@@ -44,7 +48,27 @@ runRegression p = do
   let mod' = Module "mod1" "mod-admin-keyset" (Meta Nothing Nothing) "code" (H.hash "code") mempty
   _writeRow pactdb Write Modules "mod1" mod' v
   assertEquals' "module write" (Just mod') $ _readRow pactdb Modules "mod1" v
-  assertEquals' "hash of commit 3" (-8810669039792530227) (hashWithSalt 1 <$> commit v)
+  assertEquals' "result of commit 3"
+    [ TxLog "SYS_keysets" "ks1" $
+      object [("pred" .= String "predfun")
+             ,("keys" .= [String "skdjhfskj"])]
+     ,TxLog "SYS_modules" "mod1" $
+       object [("hash" .= String "bf5fda6cead20c9349f8a7f0052ec6039bf8b38c4507db2142cbc5f2a01169941e3f0d7c7aaa3c97d53c36e63502f47d3b8c3948cce15a919055e5550f86c3ba")
+              ,("blessed" .= ([]::[Text]))
```

Comment:
Why is this one `:: [Text]` but the rest are `String`?

Reply:
Just making the TC happy (it just needs a ToJSON for the a in `[a]`)